hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContractRevision.php

```diff
@@ -315,19 +315,6 @@ private static function buildEntityQueryArray($entity, $revision, $query) {
         ";
         break;
 
-      case 'pension':
```

Comment:
don't we need to replace this with something else (fetch the pension provider name from contact)?